terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -208,7 +208,7 @@ module "glue_hive_table_creation_job" {
   execution_class             = "STANDARD"
   worker_type                 = "G.1X"
   number_of_workers           = 2
-  max_concurrent              = 1
+  max_concurrent              = 64
```

Comment:
This job takes the domain configuration as an argument so multiple concurrent runs is required.